lamindb_setup/core/_settings_storage.py

```diff
@@ -235,6 +236,12 @@ def root(self) -> UPath:
         if self._root is None:
             # below makes network requests to get credentials
             self._root = create_path(self._root_init, access_token=self.access_token)
+        elif getattr(self._root, "protocol", "") == "s3":
+            # this is needed to be sure that the root always have nonexpired credentials
```

Comment:
to fix: has

Reply:
So, this cannot yet be merged? 🤔

Reply:
Not it can be.

Reply:
Great, in that case I'll merge and start playing with it!